webpack/assets/javascripts/react_app/common/Tour/Tour.js

```diff
@@ -0,0 +1,86 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import Tour from 'reactour';
+import { noop } from '../helpers';
+import TourButton from './TourButton';
+
+const FOREMAN_COLOR = '#00759A';
+
+export const wrapComponentWithTour = (WrappedComponent, steps, id) => {
+  const tour = class TourComponent extends Component {
+    constructor(props) {
+      super(props);
+      const { alreadySeen, registerTour } = this.props;
+      if (!alreadySeen) {
+        registerTour(id);
+      }
+      this.addDismissButton();
+    }
+
+    addDismissButton = () => {
+      const lastStep = steps.length - 1;
+      const lastStepContent = steps[lastStep].content;
+      steps[lastStep].content = () => (
+        <React.Fragment>
+          {lastStepContent}
+          <TourButton id={id} />
+        </React.Fragment>
+      );
+    };
+
+    isOpen = () => {
+      // TODO: move to a selector
+      const { alreadySeen, activeTour } = this.props;
+      const inSession = sessionStorage.getItem(`TOUR_${id}`);
```

Comment:
Something is wonky here.  I notice the Run Tour button only works once, and then won't work again until I delete the value from sessionStorage.